src/openforms/prefill/contrib/family_members/filters.py

```diff
@@ -0,0 +1,133 @@
+from datetime import datetime
+from typing import Literal
+
+from openforms.contrib.haal_centraal.clients import get_brp_client
+from openforms.contrib.haal_centraal.clients.brp import NaturalPersonDetails
+from openforms.contrib.haal_centraal.constants import BRPVersions
+from openforms.utils.date import get_today
+
+from .constants import HC_DECEASED_ATTRIBUTES
+
+
+def exclude_deceased_for_haal_centraal(
```

Comment:
the function name creates the impression that you'd be filtering on already retrieved results, but this function is actually making the API calls. That's a bit unexpected.